fvtest/porttest/si.cpp

```diff
@@ -2343,3 +2343,37 @@ TEST(PortSysinfoTest, sysinfo_cgroup_get_memlimit)
 	reportTestExit(OMRPORTLIB, testName);
 	return;
 }
+
+/**
+ * Test omrsysinfo_get_processor_description.
+ */
+TEST(PortSysinfoTest, sysinfo_test_sysinfo_get_processor_description)
```

Comment:
Unfortunately, gtest style guidelines say you should not use underscores in test names (it's in the [FAQ](https://github.com/google/googletest/blob/master/googletest/docs/faq.md#why-should-test-suite-names-and-test-names-not-contain-underscore)). Further, this test name is super verbose. A better test name would be:
```c++
TEST(PortSysinfoTest, GetProcessorDescription)
```